curriculum/challenges/english/16-the-odin-project/top-the-box-model/the-box-model-question-g.md

```diff
@@ -0,0 +1,52 @@
+---
+id: 64a6702e7a7d00f97013ed0e
+title: The Box Model Question G
+challengeType: 15
+dashedName: the-box-model-question-g
+--- 
+# --description--
+
+[lines](https://cdn.statically.io/gh/TheOdinProject/curriculum/main/foundations/html_css/css-foundations/the-box-model/imgs/odin-lined.png)
+
+OK, so there might be some circles in the above image… but when it comes to layout, they fit together like rectangular boxes and not circles. In the end, laying out a webpage and positioning all its elements is deciding how you are going to nest and stack these boxes.
+
+The only real complication here is that there are many ways to manipulate the size of these boxes, and the space between them, using padding, margin, and border. The assigned articles go into more depth on this concept, but to sum it up briefly:
+
+- `padding` increases the space between the border of a box and the content of the box.
+- `margin` increases the space between the borders of a box and the borders of adjacent boxes.
+- `border` adds space (even if it’s only a pixel or two) between the margin and the padding.
+
+Be sure to study the diagrams carefully.
+
+[the box model](https://cdn.statically.io/gh/TheOdinProject/curriculum/main/foundations/html_css/css-foundations/the-box-model/imgs/box-model.png)
```

Comment:
```suggestion
![the box model](https://cdn.statically.io/gh/TheOdinProject/curriculum/main/foundations/html_css/css-foundations/the-box-model/imgs/box-model.png)
```

This too. Also, we should not rely on text within images to explain concepts, as we can't translate it.

We should extract the text from the pink box and have it copied in the challenge text.